src/org/usfirst/frc4904/robot/subsystems/GearIO.java

```diff
@@ -1,25 +1,30 @@
 package org.usfirst.frc4904.robot.subsystems;
 
 
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.OverridableCommand;
 import org.usfirst.frc4904.standard.commands.Idle;
 import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
 import org.usfirst.frc4904.standard.subsystems.motor.Motor;
 import edu.wpi.first.wpilibj.DoubleSolenoid;
 import edu.wpi.first.wpilibj.command.Subsystem;
 
-public class GearIO extends Subsystem {
+public class GearIO extends Subsystem implements OverridableCommand {
 	protected final Motor intakeRoller;
 	protected final DoubleSolenoid gullWings;
 	protected final DoubleSolenoid ramp;
 	protected GearState currentState;
 	protected RampState currentRampState;
+	public boolean rampOverride;
 
 	public GearIO(Motor intakeRoller, DoubleSolenoid gullWings, DoubleSolenoid ramp) {
 		this.intakeRoller = intakeRoller;
 		this.gullWings = gullWings;
 		this.ramp = ramp;
 		setState(GearState.TRANSPORT);
-		setRampState(RampState.EXTENDED);
+		if (RobotMap.Component.gearIO.isNotOverridden()) {
```

Comment:
This is within geario... why access the one in RobotMap?
Also we're never overriden when we construct